server/server.go

```diff
@@ -887,3 +876,25 @@ func sortedFrontendNamesForConfig(configuration *types.Configuration) []string {
 	sort.Strings(keys)
 	return keys
 }
+
+func (server *Server) configureFrontends(frontends map[string]*types.Frontend) {
+	for _, frontend := range frontends {
+		// default endpoints if not defined in frontends
+		if len(frontend.EntryPoints) == 0 {
+			frontend.EntryPoints = server.globalConfiguration.DefaultEntryPoints
+		}
+	}
+}
+
+func configureBackends(backends map[string]*types.Backend) {
```

Comment:
I can turn it into a receiver method if you really want to, but it would carry the wrong message to the reader (and my IDE's "find usage" function) that `server` is somehow needed for this method to work properly, which it isn't. It also makes things like testing more complicated.

Effectively, it'd be similar to passing an unused variable to a function call.